sample/Yarp/appsettings.json

```diff
@@ -34,9 +34,13 @@
           "Path": "/proxy-app2/{**catch-all}"
         },
         "Transforms": [
-          {
-            "PathPattern": "{**catch-all}"
-          }
+            {
+                "PathPattern": "{**catch-all}"
+            },
+            {
+                "RenameHeader": "days",
+                "Set": "range"
```

Comment:
I debugged the app and I got an exception while transformation of a request.
```
   at System.Net.Http.Headers.HttpHeaderParser.ParseValue(String value, Object storeValue, Int32& index)
   at System.Net.Http.Headers.HttpHeaders.ParseAndAddValue(HeaderDescriptor descriptor, HeaderStoreItemInfo info, String value)
   at System.Net.Http.Headers.HttpHeaders.Add(HeaderDescriptor descriptor, String value)
```

I think the reason is the header "range" is a reserved header. I renamed it and transformation worked correct.

Can you fix it please.

Reply:
You are correct. Range is a reserved header. I used `x-header` instead. No need to get fancy in an example.